sdk/eventhub/azure-eventhub/azure/eventhub/_configuration.py

```diff
@@ -10,15 +10,15 @@
     from urllib.parse import urlparse
 
 from uamqp.constants import TransportType, DEFAULT_AMQPS_PORT, DEFAULT_AMQP_WSS_PORT
-from ._retry import RetryMode
+from azure.core.pipeline.policies import RetryMode
 
 
 class Configuration(object):  # pylint:disable=too-many-instance-attributes
     def __init__(self, **kwargs):
         self.user_agent = kwargs.get("user_agent")  # type: Optional[str]
         self.retry_total = kwargs.get("retry_total", 3)  # type: int
         self.max_retries = self.retry_total  # type: int
-        self.retry_mode = kwargs.get("retry_mode", RetryMode.EXPONENTIAL)
+        self.retry_mode = RetryMode(kwargs.get("retry_mode", 'exponential'))
```

Comment:
Will this work if someone _is_ passing in the Core RetryMode enum?
e.g. `RetryMode(RetryMode.Fixed))`

Reply:
yes it will - just tested!

Reply:
added a test to validate this

Reply:
Awesome thanks!